tests/lib/rules/order-in-controllers.js

```diff
@@ -223,5 +223,37 @@ eslintTester.run('order-in-controllers', rule, {
         line: 3,
       }],
     },
+    {
+      code: `export default Controller.extend({
+        queryParams: [],
+        currentUser: service(),
+      });`,
+      output: `export default Controller.extend({
+        currentUser: service(),
+      queryParams: [],
```

Comment:
looks like indentation is broken here?

Reply:
Yes, I did not fix that, because the `[indent]` rule can do that. This is indented wrong because `currentUser` used to be the last property, so this is the whitespace that was copied. I didn't want to add another `if` statement to fix that, because people can have different indentation settings.